pysensu_yelp/__init__.py

```diff
@@ -146,7 +165,7 @@ def send_event(name, runbook, status, output, team, page=False, tip=None, notifi
     json_hash = json.dumps(result_dict)
     sock = socket.socket()
     try:
-        sock.connect(SENSU_ON_LOCALHOST)
+        sock.connect(sensu_host, sensu_port)
```

Comment:
Will this work? I think socket.connect() takes one arg - (addr, port) tuple, not 2 separate args.